crates/e2e-test-utils/src/testsuite/mod.rs

```diff
@@ -138,6 +140,16 @@ impl<I: 'static> TestBuilder<I> {
         LocalPayloadAttributesBuilder<N::ChainSpec>: PayloadAttributesBuilder<
             <<N as NodeTypes>::Payload as PayloadTypes>::PayloadAttributes,
         >,
+        <<N as Node<TmpNodeAdapter<N>>>::AddOns as NodeAddOns<Adapter<N>>>::Handle:
+            RpcHandleProvider<
+                NodeAdapter<
+                    TmpNodeAdapter<N>,
+                    <<N as Node<TmpNodeAdapter<N>>>::ComponentsBuilder as NodeComponentsBuilder<
+                        TmpNodeAdapter<N>,
+                    >>::Components,
+                >,
+                <<N as Node<TmpNodeAdapter<N>>>::AddOns as RethRpcAddOns<Adapter<N>>>::EthApi,
+            >,
```

Comment:
think we can clean the code a bit if we make a type alias. Similar to there:
https://github.com/scroll-tech/reth/blob/24f1972a5896b7ad35196f4e7e3a1cf9dc84dd72/crates/payload/basic/src/lib.rs#L47